### PR TITLE
Include PHP Min Version code

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#116283](http://central.tri.be/issues/116283)
---
📹 http://i.bordoni.me/1bf0b5baf223

**Before updating addons**
screen shot 2019-01-17 at 01 16 37 am

**After all plugins require correct version**
screen shot 2019-01-17 at 01 18 21 am

https://github.com/moderntribe/the-events-calendar/pull/2381 https://github.com/moderntribe/event-tickets/pull/1064 https://github.com/moderntribe/events-community-tickets/pull/131 https://github.com/moderntribe/events-community/pull/369 https://github.com/moderntribe/event-tickets-plus/pull/682 https://github.com/moderntribe/events-pro/pull/1109 https://github.com/moderntribe/events-eventbrite/pull/278 https://github.com/moderntribe/events-filterbar/pull/160

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [php-min-version.php:57](https://github.com/moderntribe/event-tickets/blob/ce1a6b164bcbe3f627ca95c56f5c20adc2d8338f/src/functions/php-min-version.php#L57)

> via jenkins/codesniffer-phpcs

File: src/functions/php-min-version.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ *
+ * @return string
+ */
+function tribe_tickets_not_php_version_message() {
+	return wp_kses_post( sprintf(
+			__( '<b>Event Tickets</b> requires PHP %1$s or higher, and the plugin has now disabled itself.', 'event-tickets' ),
+			tribe_tickets_get_php_min_version()
+		) ) .
+		'<br />' .
+		esc_attr__( 'To allow better control over dates, advanced security improvements and performance gain.', 'event-tickets' ) .
```
Comment (sc0ttkclark) [inline src/functions/php-min-version.php]:
```suggestion
		esc_html__( 'To allow better control over dates, advanced security improvements and performance gain.', 'event-tickets' ) .
```

File: src/functions/php-min-version.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		'<br />' .
+		esc_attr( sprintf(
```
Comment (sc0ttkclark) [inline src/functions/php-min-version.php]:
```suggestion
		esc_html( sprintf(
```

File: src/functions/php-min-version.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+			__( 'Contact your Hosting or your system administrator and ask for this Upgrade to version %1$s of PHP.', 'event-tickets' ),
+			tribe_tickets_get_php_min_version()
+		) );
+}
+
+/**
+ * Echos out the error for the PHP min version as a WordPress admin Notice
```
Comment (juanfra) [inline src/functions/php-min-version.php]: Echoes

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [php-min-version.php:57](https://github.com/moderntribe/event-tickets/blob/6e28411ab8a0cfa627f8cbb2d197492ed9df8a0e/src/functions/php-min-version.php#L57)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [php-min-version.php:57](https://github.com/moderntribe/event-tickets/blob/9d966b599138bcfbeef4196373805001afc85e5f/src/functions/php-min-version.php#L57)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [php-min-version.php:103](https://github.com/moderntribe/event-tickets/blob/f64b3c533ccff74a1ee9925517b0de71d557cc2e/src/functions/php-min-version.php#L103)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [php-min-version.php:103](https://github.com/moderntribe/event-tickets/blob/23ecc189d41fc2d8544b5c92af990036ca168ec8/src/functions/php-min-version.php#L103)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [php-min-version.php:103](https://github.com/moderntribe/event-tickets/blob/e95f5cd4af20c0f4e9f31fd62e7a5bf46303d27a/src/functions/php-min-version.php#L103)

> via jenkins/codesniffer-phpcs

**Pull request merged**